metapackages/auto-instrumentations-node/src/utils.ts

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { diag } from '@opentelemetry/api';
+import {
+  DnsInstrumentation,
+  DnsInstrumentationConfig,
+} from '@opentelemetry/instrumentation-dns';
+import {
+  ExpressInstrumentation,
+  ExpressInstrumentationConfig,
+} from '@opentelemetry/instrumentation-express';
+import { GrpcInstrumentationConfig } from '@opentelemetry/instrumentation-grpc/build/src/types';
```

Comment:
Why is this one different than all the others

Reply:
Can we fix this so they are all the same?

Reply:
leaving for now the basic config, can be changed after releasing the core as grpc is part of core :/